client/src/components/OpeningTimes.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function OpeningTimes({ currentStation }) {
+  const convertOpeningTimes = () => {
+    // console.log(`Test: ${Object.keys(currentStation.openingtimes_json.openingTimes[0])}`);
+    if (Object.keys(currentStation.openingtimes_json).length === 0) {
+      return (
+        <p>Uns liegen leider keine Informationen zu den Öffnungzeiten dieser Tankstelle vor</p>
+      );
+    } else return <p>Hier liegen Tankstellenpreise vor.</p>;
+  };
+
+  return <div>{convertOpeningTimes()}</div>;
```

Comment:
The function convertOpeningTimes could be inlined:
`<p>{ Object.keys(...).length === 0 ? "..." : "..." }</p> `